cuda_bindings/tests/test_nvjitlink.py

```diff
@@ -39,6 +39,8 @@

```

Comment:
I am a little uncomfortable checking in a binary blob like this. How about we keep the (C++) source code and generate needed PTX/LTOIR on the fly, using NVRTC? It also avoids us hard-coding the PTX version in the blob, because the PTX version will be determined by the NVRTC version seen at run time (which hopefully is installed properly).

Reply:
agreed. fixed